pkg/scheduler/internal/cache/cache.go

```diff
@@ -708,6 +710,24 @@ func (cache *cacheImpl) addNodeImageStates(node *v1.Node, nodeInfo *framework.No
 	nodeInfo.ImageStates = newSum
 }
 
+// updateNodeImageStates update node record from image entries having the node
+// in imageStates cache when receive node add/update/delete events.
+func (cache *cacheImpl) updateNodeImageStates(newNode *v1.Node) {
```

Comment:
maybe this is a method that is called inside of `addNodeImageStates` or `removeNodeImageStates` as the second step to update other nodes' `imageState`?